evm_loader/cli/src/main.rs

```diff
@@ -1734,12 +1822,42 @@ fn main() {
                 let lamports = value_t_or_exit!(arg_matches, "lamports", u64);
                 let space = value_t_or_exit!(arg_matches, "space", u64);
 
-                command_create_ether_account(&config, &ether, lamports, space)
+                let token_mint = pubkey_of(arg_matches, "token_mint")
+                    .unwrap_or_else(|| {
+                        let elf_params = read_elf_parameters_from_account(&config).unwrap();
+                        Pubkey::from_str(elf_params.get("NEON_TOKEN_MINT").unwrap()).unwrap()
+                    });
+
+
+                command_create_ether_account(&config, &ether, lamports, space, &token_mint)
             }
             ("deploy", Some(arg_matches)) => {
                 let program_location = arg_matches.value_of("program_location").unwrap().to_string();
 
-                command_deploy(&config, &program_location)
+                let mut elf_params : HashMap<String,String> = HashMap::new();
+
+                let token_mint = pubkey_of(arg_matches, "token_mint")
+                    .unwrap_or_else(|| {
+                        elf_params = read_elf_parameters_from_account(&config).unwrap();
+                        Pubkey::from_str(elf_params.get("NEON_TOKEN_MINT").unwrap()).unwrap()
+                    });
+
+                let collateral_pool_base = pubkey_of(arg_matches, "collateral_pool_base")
+                    .unwrap_or_else(|| {
+                        if elf_params.is_empty(){
```

Comment:
I suggest moving this logic inside the `get_elf_param` function, which will read elf parameters and cache it for the first call.

Reply:
Created #446 446.